src/deploy-tes-on-azure/Configuration.cs

```diff
@@ -22,18 +21,11 @@ public class Configuration : UserAccessibleConfiguration
         public string PostgreSqlVersion { get; set; } = "11";
         public string DefaultPostgreSqlSubnetName { get; set; } = "sqlsubnet";
         public int PostgreSqlStorageSize { get; set; } = 128;  // GiB
-        public string Name { get; set; } = CreateNewName();
```

Comment:
both deployers will need their AKS templates updated for these changes

Reply:
See microsoft/CromwellOnAzure#586